tests/tensor/test_nlinalg.py

```diff
@@ -119,6 +120,47 @@ def test_matrix_dot():
     assert _allclose(numpy_sol, aesara_sol)
 
 
+def test_matmul():
+    rng = np.random.default_rng(utt.fetch_seed())
+    # test output when both inputs are matrices
+    x1 = rng.random((5, 3))
+    x2 = rng.random((3, 2))
```

Comment:
The tests are currently failing under `aesara.config.floatX = "float32"`.  This is due to the NumPy test values having type `float64` by default.  If you convert the values with `.astype(config.floatX)`, the problem should be solved.